source/Interfaces/NetWorth/client/ClientInterface.hpp

```diff
@@ -0,0 +1,161 @@
+/**
+ * This file is part of the Fall 2023, CSE 491 course project.
+ * @brief A networking interface that allows information to be sent across a  network
+ * @note Status: PROTOTYPE
+ **/
+
+#pragma once
+
+#include <cassert>
+#include <string>
+#include <vector>
+#include <SFML/Network/UdpSocket.hpp>
+#include <SFML/Network/Packet.hpp>
+#include <memory>
+
+#include "../NetworkInterface.hpp"
+#include "../../TrashInterface.hpp"
+
+using namespace sf;
+
+namespace cse491 {
+    namespace netWorth{
+    /**
+     * The interface of our client that will be interacting and connection with our server
+     */
+        class ClientInterface {
+        private:
+            std::unique_ptr<TrashInterface> mTrash;     /// Display interface
```

Comment:
You want to minimize the amount of information that you send across the network to just the bare minimum that an interface might need.  As such, you probably don't want to use a TrashInterface for the client side, but something that you build on your own (perhaps initially based on the TrashInterface) that requires only the more limited data that you decide to send it.  I think in the long term, this will be easier for you where you process down input data on the server side and send it along to the client.

All of that said, I don't see where you're actually setting up the client interface...

Reply:
I believe we should have actually removed this from our current PR, as it is not being used. The intention was that the client would have that assigned at the beginning so that it when it initially connects to the server it would be given the interface, but due to a lack of serialization at the moment we were stuck on how to perform this.